session8/src/com/itschool/homework/EX8.java

```diff
@@ -0,0 +1,11 @@
+package com.itschool.homework;
+
+public class EX8 {
+
+    public static void main(String[] args) {
+
+        StringBuilder stringBuilder = new StringBuilder("Sunny day");
+        stringBuilder.append(" or cloudy?");
+        System.out.println("Updated string: " + stringBuilder.toString());
```

Comment:
again, this seems different than the exercise's request:
_Write a Java program that inserts a given string into another string at a specified index using the StringBuilder class._
Could be possible that you solved other exercises and named them incorrectly?